### ISSUE TITLE
Adding more blocks

### ISSUE THREAD

**Issue by Pantoufleee** [Dorky106/RiseoftheNewColonies#1]
White / black / brown blocks (stairs, corners etc ...)
Same but for log (every type of logs if possible)

Thanks for making buildings easier man !

**Comment by Dorky106** [comment]
So different coloured stone versions?

**Comment by Pantoufleee** [comment]
Yes and if possible, same for marble blocks from the settlers mod =)

**Comment by Dorky106** [comment]
I'll have to grab the texture for it.

**Comment by Dorky106** [comment]
version 0.0.7.0 is going to break all of the wooden decor blocks.
I plan on changing the name to planks{block}, as I want to add the other types of planks as well.
